src/rust/ide/json-rpc/src/handler.rs

```diff
@@ -131,11 +134,16 @@ pub type OngoingCalls = HashMap<Id,oneshot::Sender<ReplyMessage>>;
 
 pub use shapely::shared;
 
+/// The default timeout for all responses given in seconds.
+const TIMEOUT_SECS:u64 = 1;
```

Comment:
I'm afraid that not but I have to ask — perhaps `Duration` supports `const` so we could properly type this?
(if not, leave as-is)